src/test/regress/expected/multi_extension.out

```diff
@@ -899,6 +899,14 @@ SELECT * FROM multi_extension.print_extension_changes();
                  | function worker_fix_partition_shard_index_names(regclass,text,text) void
 (3 rows)
 
+-- Use a synthetic pg_dist_shard record to show that upgrade fails
+-- when there are cstore_fdw tables
+INSERT INTO pg_dist_shard (logicalrelid, shardid, shardstorage) VALUES ('pg_dist_shard', 1, 'c');
+ALTER EXTENSION citus UPDATE TO '11.0-1';
+ERROR:  cstore_fdw tables are deprecated as of Citus 11.0
+HINT:  Convert your cstore_fdw tables to the columnar access method before upgrading
```

Comment:
At this point the citus 11 binaries are already loaded right? Do you see any problems with that situation? for example, can they follow https://github.com/citusdata/cstore_fdw#important-notice-columnar-storage-is-now-part-of-citus these steps at this point?

Reply:
They'd have to reinstall the older packages.

Reply:
So, maybe we should HINT that?

Reply:
> So, maybe we should HINT that?

(+1 for that)